MusicCat-Console/Program.cs

```diff
@@ -52,5 +64,40 @@ static void Main(string[] args)
 			logStream?.Dispose();
             Listener.Stop();
         }
+
+		private static int counter;
+		private static void OnConfigChange(object sender, FileSystemEventArgs e)
+		{
+			if (counter == 0)
+			{
+				counter++;
+				return;
+			}
+
+			counter = 0;
+			try
+			{
+				logWriter?.Dispose();
+				logStream?.Dispose();
+				Listener.Config = Config.ParseConfig(out logStream, out logWriter);
+			}
+			catch (Exception ex)
+			{
+				Log(new ApiLogMessage($"Failed to de-serialize config, using default config instead. Exception: {ex.Message}{Environment.NewLine}{ex.StackTrace}", ApiLogLevel.Warning));
+				Listener.Config = Config.DefaultConfig;
+			}
+			Listener.Stop();
+			Listener.Start();
+			try
+			{
+				MetadataStore.SongList.Clear();
+				MetadataStore.LoadMetadata(Log);
+			}
+			catch (Exception ex)
+			{
+				Log(new ApiLogMessage($"Failed to load metadata. Exception: {ex.Message}{Environment.NewLine}{ex.StackTrace}", ApiLogLevel.Critical));
+				Environment.Exit(1);
```

Comment:
Instead of quitting, fall back to old metadata. The app should only quit if it fails to load any metadata on startup. Otherwise it should always keep going with the last good data it has.